src/Fixer/AbstractPhpUnitFixer.php

```diff
@@ -69,19 +70,21 @@ final protected function getDocBlockIndex(Tokens $tokens, int $index): int
 
     /**
      * @param list<string> $preventingAnnotations
+     * @param list<string> $preventingAttributes
```

Comment:
I believe this should be:

```suggestion
     * @param list<class-string> $preventingAttributes
```